src/settings.c

```diff
@@ -41,6 +41,14 @@ static void on_sidebar_pos_left_changed(GSettings *settings, gchar *key, gpointe
 }
 
 
+static void on_editor_font_changed(GSettings *settings, gchar *key, gpointer user_data)
+{
```

Comment:
@codebrainz do we need an `on_foo_bar_setting_changed()` for every "interesting" setting? Or can we use one `on_setting_changed()` with `user_data` specifying which setting.  Then we would only possibly need a few callbacks.